src/BuildScriptGeneratorCli/Commands/OptionTemplates.cs

```diff
@@ -20,5 +20,6 @@ internal static class OptionTemplates
         public const string Property = "-p|--property <key-value>";
         public const string ManifestDir = "--manifest-dir <directory-path>";
         public const string AppType = "--apptype <application-type>";
+        public const string DynamicInstallRootDir = "--dynamic-install-root-dir <dir-path>";
```

Comment:
DynamicInstallSdkDir instead DynamicInstallRootDir? IMO keeping root in the name may be confusing

Reply:
I wanted to mention it explicitly because there are some cases where users might want to install it in a way which benv and oryx does not understand, for example installing at /usr/bin. We cannot support it currently since we expect certain structure for our platforms.